lib/beaker/dsl/structure.rb

```diff
@@ -54,6 +55,51 @@ def step step_name, &block
         end
       end
 
+      # Provides a method to help manual tests. So we can use beaker to set up
+      # the environment, then prompt a user to manually check the setup.
+      # @param [String] step_name The name of the step to be logged.
+      # @param [String] fail_message The message to return if this manual step failed
+      def manual_step step_name, fail_message='Manual step failed.'
+        logger.notify "\n* #{step_name}\n"
+        if(!@options.has_key?(:exec_manual_tests))
+          # if the option -exec-manual-tests is not set then this has executed outside of a manual tests
+          # so we raise an error to avoid issues
+          raise('--exec-manual-tests option not set, this means a manual_step was used outside a manual_test')
+        end
+
+        set_current_step_name(step_name)
+        # Here we prompt the user to tell us if the step passed or failed
+        loop do 
+          input = Readline.readline('did this step pass, Y/n?', true).squeeze(" ").strip
+          if input == 'Y'
+            break
+          elsif input == 'n'
+            raise Beaker::DSL::FailTest, fail_message
```

Comment:
@Andrew-Hayes we were debating whether or not having a `binding.pry` here would be better than this process, & I think we ended up being OK with using `readline` instead, but we're lacking some context to be confident of this.

It did seem to us that if you had a lot of testing to run, and some-odd number of manual tests to run as a part of that where you had to do visual comparison for verification, it could be difficult to juggle all of the reasons that things failed in your head as failures pile up. We were thinking a fail workflow like this might be better:
```
did this step pass, Y/n? n
default failure message: <fail_message argument>
to use this failure message, press <char>. Otherwise, enter a new failure message:
```
before, we'd talked about how juggling code to change vs code to test can be a pain for QA (I assume the `fail_message` parameter can fit in this boat).

Is this a case that comes up for you guys? We're curious about what your manual testing environment looks like, so that we can better match our reviews up with what you're really doing.